README.md

```diff
@@ -75,6 +59,9 @@ The error is normal because the default command is designed for child containers
 FROM openjdk
 ADD my_app_0.0.1.jar app.jar
 ```
+# Devlopment Guide
+
+* See [DEVELOPING.md](DEVELOPING.md)
```

Comment:
Maybe say, "See [instructions](DEVELOPING.md) on how to build the image from source."?